lib/active_shipping/carriers/ups.rb

```diff
@@ -631,6 +631,14 @@ def build_international_forms(xml, origin, destination, packages, options)
       end
     end
 
+    def products(packages)
+      if packages && packages.first && packages.first.options && packages.first.options[:products]
+        packages.first.options[:products]
```

Comment:
Is `.first` good enough or should we combine all products in all packages?

Reply:
@brian31988 ^^^ Was this comment resolved IRL? Is `packages` always an array with one element?

Reply:
Yea packages is always an array with one element